prime-router/metadata/schemas/covid-19.schema

```diff
@@ -872,7 +872,8 @@ elements:
 
   - name: test_result_status
     type: CODE
-    default: F
+    cardinality: ZERO_OR_ONE
+# Don't default.    default: F
     valueSet: hl70123
```

Comment:
I think having a default here is not a good idea.   User should always tell us if a value is Final or Corrected or whatever.  We can't know that.